addons/stock/models/stock_inventory.py

```diff
@@ -100,12 +106,19 @@ def _selection_filter(self):
     def onchange_filter(self):
```

Comment:
What is this onchange ? The original one was unreadable, this one is leveled up in its category.